sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="NUnit" />
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="OpenTelemetry" VersionOverride="1.4.0" />
```

Comment:
Instead of using VersionOverride, add this to [package.data.props](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L185) instead in the tests section.